### PR TITLE
New HOW-TO guide: how to add extra axes to a diffractometer

### LINKED ISSUE #46: DOC: describe how to add extra motors to a diffractometer class
See https://github.com/APS-4ID-POLAR/ipython-polar/issues/39 and add documentation for this typical use case.

### PR DESCRIPTION
**Pull request by prjemian**
fix #46

### CODE REVIEW

**Comment by prjemian** [comment]
@mrakitin ready for review

**Review by mrakitin — APPROVED** [review]
Looks good. I have a single comment to clarify the units and limits for the `spinner` object.

File: examples/how_extra_real_axis.ipynb [hunk 1]
```diff
@@ -0,0 +1,439 @@
...
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Customize the subclass\n",
+    "\n",
+    "Following a pattern, we simply add a *spinner* motor to the class and create a new diffractometer object.  Here's how:\n",
+    "\n",
+    "```python\n",
+    "class EnhancedFourCircle(FourCircle):\n",
+    "    spinner = Component(SoftPositioner, kind=\"hinted\", limits=(-3600, 3600), init_pos=0)\n",
```
[the remark this answers is not in the document]
Comment (prjemian) [reply examples/how_extra_real_axis.ipynb:235]: Completely arbitrary values and numbers.  Should state in the preceding text.

[the remark this answers is not in the document]
Comment (prjemian) [reply examples/how_extra_real_axis.ipynb:235]: Ah, there *is* an `egu` kwarg.

[the remark this answers is not in the document]
Comment (prjemian) [reply examples/how_extra_real_axis.ipynb]: Setting `egu="rotations"` and changing limits to +/- 10,000 rotations.  It's a spinner; it's gotta spin!

**Pull request merged**